be/src/common/config.cpp

```diff
@@ -908,6 +908,8 @@ DEFINE_Validator(file_cache_type, [](std::string_view config) -> bool {
     return config.empty() || config == "file_block_cache";
 });
 
+DEFINE_String(tmp_file_dir, "tmp");
+
```

Comment:
<img width="993" alt="image" src="https://github.com/apache/doris/assets/6919662/e596b87e-997e-4326-9e33-f179dcdbbed3">

it seems not compatible

Reply:
not using tmp file manager now, just use storage root path tmp directory as temporary writing dir